crates/index-scheduler/src/error.rs

```diff
@@ -103,7 +104,7 @@ pub enum Error {
     )]
     InvalidTaskCanceledBy { canceled_by: String },
     #[error(
-        "{index_uid} is not a valid index uid. Index uid can be an integer or a string containing only alphanumeric characters, hyphens (-) and underscores (_), and can not be more than 512 bytes."
+        "{index_uid} is not a valid index uid. Index uid can be an integer or a string containing only alphanumeric characters, hyphens (-) and underscores (_), and can not be more than 400 bytes."
```

Comment:
Why 400 bytes?
Shouldn't it be 511 like the pk since it's stored as an lmdb key?

Reply:
this is what [the code is currently checking](https://github.com/meilisearch/meilisearch/blob/exactly-512-bytes-docid-fails/crates/dump/src/reader/v4/meta.rs#L43), so I'm just fixing the error message in that way.

Talking about it with @Kerollmops, the rationale was that at some point we might have wanted to include more things than just the index uid in the db key, or to include it with additional data as part of the key in another db.

I don't mind the 400 bytes limit, as that's already a super long index name. I mean `"Supercalifragilisticexpialidocious index with uid 30f61e7b-db93-4c76-937c-baeb5fa85a54 30f61e7b-db93-4c76-937c-baeb5fa85a54 30f61e7b-db93-4c76-937c-baeb5fa85a54` is only 160 bytes.